### ISSUE TITLE
MarkdownUI is not usable inside LazyVStack

### ISSUE THREAD

**Issue by Lyncore** [gonzalezreal/swift-markdown-ui#186]
When i'm trying to use MarkdownUI in LazyVStack inside a ScrollView, layout jumping on scrolling up

**Steps to reproduce**
```
ScrollView(.vertical, showsIndicators: false) {
                LazyVStack(spacing: 8) {
                    ForEach(result, id: \.id) { item in
                        VStack(alignment: .leading) {
                            HStack(alignment: .top) {
                                Text(item.title).font(.title3)
                                Spacer()
                                Text(item.createdAt).foregroundColor(Color.accentColor)
                            }
                            Markdown(item.content).markdownTheme(Theme()
                                    .link {
                                        ForegroundColor(Color.accentColor)
                                    }
                                    .text {
                                        ForegroundColor(Color.textSecondaryColor)
                                    }

                            )
                        }
                                .padding(8)
                                .background(Color.primaryColor)
                                .clipShape(RoundedRectangle(cornerRadius: 12, style: .continuous))
                                .padding(.horizontal, 8)
                    }
                }
                        .padding(.vertical, 8)
            }
```

**Version information**
- MarkdownUI: 2.0.0
- OS: iOS 15
- Xcode: 14.2.0

**Comment by gonzalezreal** [comment]
Hi @Lyncore,

Could you please provide more information? For example, how the Markdown you're trying to render looks like? Can you create a sample project or provide a video capture?

**Comment by Lyncore** [comment]
I reproduced the bug by creating this code: [MarkdownNewsApp.swift](https://gist.github.com/Lyncore/63c4bf8c07d97a473695f2a47267719e)

Tested on iOS 15 and iOS 16, always have the same result
You need to scroll down to 20-30 elements, then when scrolling up, the elements begin to jumping

**Comment by gonzalezreal** [comment]
Thanks for providing more context. I can only reproduce the issue when the Markdown content has images.
I found an issue in the code that caches and loads the images and fixed it here: #187.

Please check if #187 fixes the issue for you.

Here are a couple of suggestions about your code (not related to this issue):
- If you apply the same theme to all items in the list, you should call the `markdownTheme` modifier outside the `ForEach` view. You can even place the modifier below the scroll view. This will make your code more performant.
- Check the [documentation](https://swiftpackageindex.com/gonzalezreal/swift-markdown-ui/2.0.0/documentation/markdownui/gettingstarted) on how to style the Markdown content. You can use the `markdownTextStyle` modifier instead of replacing the whole theme using `markdownTheme`.

**Comment by gonzalezreal** [comment]
Hi @Lyncore,

I added a demo on how to use a `Markdown` view with image contents inside a `LazyVStack` and avoid re-layouts when scrolling up:
https://github.com/gonzalezreal/swift-markdown-ui/blob/5e15c0d75e42b9840301503e0df681027d4bea05/Examples/Demo/Demo/LazyLoadingView.swift

I hope this helps. If you have questions, please open a discussion instead of an issue, as this is not an issue per se.
